actors/builtin/miner/miner_state.go

```diff
@@ -259,6 +269,151 @@ func (st *State) ComputeProvingSet(store adt.Store) ([]abi.SectorInfo, error) {
 	return sectorInfos, nil
 }
 
+func computeVestAmount(total abi.TokenAmount, cliffStartEpoch abi.ChainEpoch, fullyVestedEpoch abi.ChainEpoch, quantizedEpoch abi.ChainEpoch) abi.TokenAmount {
+	// PARAM_FINISH linear vesting placeholder
+	numVestWindows := (fullyVestedEpoch - cliffStartEpoch) / VestIncrement
+	_ = (quantizedEpoch - cliffStartEpoch) / VestIncrement // number of VestIncrement elapsed for other vest functions
+	quantizedVestAmount := big.Div(total, big.NewInt(int64(numVestWindows)))
+	return quantizedVestAmount
```

Comment:
So the result here now appears not to depend on the current epoch, this is just a "constant" return value of the total divided by the number of vesting stages. But the number of vesting stages depends on the total length of the vesting period. Is that right?

This integer division will also truncate and short-change the miner if the total is not an exact multiple of the windows. You can't use a function like this to compute a fixed value to vest at each point.